docker/build_scripts/install-libxcrypt.sh

```diff
@@ -19,21 +19,24 @@ check_sha256sum "v${LIBXCRYPT_VERSION}.tar.gz" "${LIBXCRYPT_HASH}"
 tar xfz "v${LIBXCRYPT_VERSION}.tar.gz"
 pushd "libxcrypt-${LIBXCRYPT_VERSION}"
 ./autogen.sh > /dev/null
+
```

Comment:
`libxcrypt` shall not be installed on musllinux images. c.f. https://github.com/pypa/auditwheel/pull/315#discussion_r680390843